src/MainApp.jsx

```diff
@@ -20,6 +22,9 @@ registerIcons();
 
 const MainApp = () => (
   <AppProvider store={configureStore()}>
+    <Helmet>
+      <link rel="shortcut icon" href={getConfig().FAVICON_URL} type="image/x-icon" />
+    </Helmet>
```

Comment:
Should the favicon be removed from here then: https://github.com/openedx/frontend-app-authn/blob/e2899e66d107ef1625b7c6a971f032a71822d634/public/index.html#L7